Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/ParametersSectionTranslator.swift

```diff
@@ -20,15 +20,27 @@ struct ParametersSectionTranslator: RenderSectionTranslator {
         translateSectionToVariantCollection(
             documentationDataVariants: symbol.parametersSectionVariants
         ) { _, parameters in
-            guard !parameters.parameters.isEmpty else { return nil }
+            guard !(parameters.parameters.isEmpty || parameters.parameters.count < 1) else {
+                return ParameterRenderSection(name: "", content: []) as? RenderSection
+            }
             
             return ParametersRenderSection(
                 parameters: parameters.parameters
                     .map { parameter in
                         let parameterContent = renderNodeTranslator.visitMarkupContainer(
                             MarkupContainer(parameter.contents)
                         ) as! [RenderBlockContent]
-                        return ParameterRenderSection(name: parameter.name, content: parameterContent)
+                        
+                        guard !parameterContent.isEmpty && parameterContent.count != 0 else {
+                            return ParameterRenderSection(name: parameter.name, content: parameterContent)
+                        }
+                        
+                        var capitalizedParameterContent = [parameterContent[0].withFirstWordCapitalized]
+                        if parameters.parameters.count > 1 {
+                            capitalizedParameterContent = [parameterContent[0].withFirstWordCapitalized] + parameterContent[1...]
+                        }
```

Comment:
I think you can simplify this a bit, because `[1...]` will return an empty array when there are zero or one elements.

```suggestion
                        let capitalizedParameterContent = [parameterContent[0].withFirstWordCapitalized] + parameterContent[1...]
```

Reply:
Yes, changed this back to what you suggested, thanks!